client/src/public/ts/index.ts

```
@@ -22,6 +22,7 @@ import {
   PopupNotificationButtonStyle
 } from "./popup-notification";
 import { changeBackgroundImage } from "./change-background-image";
+import { showUserLookupPopUp } from "./lookup-user";
 let startInitTime: number = Date.now();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'startInitTime' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)